src/main/java/uk/gov/hmcts/reform/prl/services/FeeService.java

```diff
@@ -174,13 +182,23 @@ && isFl403ApplicationAlreadyPresent(caseData)) {
         return feeType;
     }
 
+    private FeeType getFeeTypeByApplicationReason(FeeRequest feeRequest) {
+        return isc2WithOrder(feeRequest.getApplicationReason()) ? CHILD_ARRANGEMENTS_ORDER : C2_WITH_NOTICE;
+    }
+
+    public boolean isc2WithOrder(String applicationReason) {
+        return applicationReason.equals(CHILD_ARRANGEMENTS_ORDER_TO_LIVE_SPEND_TIME.getId())
```

Comment:
Please put constants first then check for equals

Reply:
address RC